index.html

```diff
@@ -277,6 +280,7 @@ <h2 class="form-title">Contact me</h2>
           </form>
         </div>
       </section>
```

Comment:
- [ **OPTIONAL** ] Also to match the Figma design I suggest you add the expected `padding` in all the `inputs` and `textearea` fields and also a clear color_text in the `textearea` field. Here is how looks your inputs once filled with some text
![Screenshot from 2022-10-11 23-30-41](https://user-images.githubusercontent.com/83042742/195202693-dfd1464e-d1c9-48b6-825d-29c2faf6e13f.png)